autopush/websocket.py

```diff
@@ -179,6 +224,7 @@ class PushState(object):
         'direct_updates',
 
         'msg_limit',
+        '_reset_uaid',
         'reset_uaid',
```

Comment:
methods/properties don't need __slot__ definitions (reset_uaid, uaid/_hash_obj, etc).

I've been meaning to remove remove pause/resume/stopProducing from this list..